scripts/ssm-build-deb.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# Execute the following as root to install lintian and fpm
+# apt-get install lintian
+# apt-get install ruby ruby-dev rubygems build-essential
+# gem install --no-ri --no-rdoc fpm
+
+# Then run this file, altering the version.
+# This file will create two versions of the deb file:
+# - apel-ssm_<version>_all.deb contains all the files necessary to run a
+#   the SSM as a sender.
+# - apel-ssm-service_<version>_amd64.deb will install service files
+#   necessary to run the SSM as a receiver on amd64 style architectures.
+# After building apel-ssm_<version>_all.deb, this script will run it
+# against lintian to highlight potential issues to the builder.
+
+set -eu
+
+TAG=2.2.0-1
+
+SOURCE_DIR=~/debbuild/source
+BUILD_DIR=~/debbuild/build
+
+# Where to install the python lib files
+PYTHON_INSTALL_LIB=/usr/lib/python2.7/dist-packages
+
+# Split the tag into version and package number
+# so they can be passed to fpm separately.
+# This will work with RCs of the form X.Y.Z-0.A.rcA
+VERSION=`echo "$TAG" | cut -d - -f 1`
+ITERATION=`echo "$TAG" | cut -d - -f 2`
+
+# Create SSM and DEB dir (if not present)
+mkdir -p $SOURCE_DIR
+mkdir -p $BUILD_DIR
+
+# Clean up any previous build
+rm -rf $SOURCE_DIR/*
+rm -rf $BUILD_DIR/*
+
+# Get and extract the source
+TAR_FILE=${TAG}.tar.gz
+TAR_URL=https://github.com/apel/ssm/archive/$TAR_FILE
+wget --no-check-certificate $TAR_URL -O $TAR_FILE
+tar xvf $TAR_FILE -C $SOURCE_DIR
+rm -f $TAR_FILE
+
+fpm -s python -t deb \
+-n apel-ssm \
+-v $VERSION \
+--iteration $ITERATION \
+-m "Apel Administrators <apel-admins@stfc.ac.uk>" \
+--description "Secure Stomp Messenger (SSM)." \
+--no-auto-depends \
+--depends python2.7 \
+--depends python-pip \
+--depends python-ldap \
+--depends libssl-dev \
+--depends libsasl2-dev \
+--deb-changelog CHANGELOG \
+--python-install-bin /usr/bin \
+--python-install-lib $PYTHON_INSTALL_LIB \
+--exclude *.pyc \
+$SOURCE_DIR/ssm-$VERSION/setup.py
+
+fpm -s pleaserun -t deb \
+-n apel-ssm-service \
+-v $VERSION \
+--iteration $ITERATION \
+-m "Apel Administrators <apel-admins@stfc.ac.uk>" \
+--description "Secure Stomp Messenger (SSM) unit files." \
```

Comment:
Unit files?

Reply:
the system files that let you do `service apel-ssm start`

Reply:
I thought there wern't any for Ubuntu.

Reply:
Renamed "unit files" to "Service Daemon files"